Change exported include headers to reference cbs/config.h to reduce chance of include conflicts.

diff --git a/libavcodec/arm/mathops.h b/libavcodec/arm/mathops.h
index dc57c5571c..972fcb101f 100644
--- a/libavcodec/arm/mathops.h
+++ b/libavcodec/arm/mathops.h
@@ -23,7 +23,7 @@
 #define AVCODEC_ARM_MATHOPS_H
 
 #include <stdint.h>
-#include "config.h"
+#include "cbs/config.h"
 #include "libavutil/common.h"
 
 #if HAVE_INLINE_ASM
diff --git a/libavcodec/mathops.h b/libavcodec/mathops.h
index 84a924f31b..b21a1852b1 100644
--- a/libavcodec/mathops.h
+++ b/libavcodec/mathops.h
@@ -26,7 +26,7 @@
 
 #include "libavutil/attributes_internal.h"
 #include "libavutil/common.h"
-#include "config.h"
+#include "cbs/config.h"
 
 #define MAX_NEG_CROP 1024
 
diff --git a/libavcodec/mips/mathops.h b/libavcodec/mips/mathops.h
index bb9dc8375a..eaf8a7b035 100644
--- a/libavcodec/mips/mathops.h
+++ b/libavcodec/mips/mathops.h
@@ -23,7 +23,7 @@
 #define AVCODEC_MIPS_MATHOPS_H
 
 #include <stdint.h>
-#include "config.h"
+#include "cbs/config.h"
 #include "libavutil/common.h"
 
 #if HAVE_INLINE_ASM
diff --git a/libavcodec/ppc/mathops.h b/libavcodec/ppc/mathops.h
index dbd714fcd4..3bdea7cd8f 100644
--- a/libavcodec/ppc/mathops.h
+++ b/libavcodec/ppc/mathops.h
@@ -24,7 +24,7 @@
 #define AVCODEC_PPC_MATHOPS_H
 
 #include <stdint.h>
-#include "config.h"
+#include "cbs/config.h"
 #include "libavutil/common.h"
 
 #if HAVE_PPC4XX
diff --git a/libavcodec/x86/mathops.h b/libavcodec/x86/mathops.h
index ca7e2dffc1..f5cf8e1d70 100644
--- a/libavcodec/x86/mathops.h
+++ b/libavcodec/x86/mathops.h
@@ -22,7 +22,7 @@
 #ifndef AVCODEC_X86_MATHOPS_H
 #define AVCODEC_X86_MATHOPS_H
 
-#include "config.h"
+#include "cbs/config.h"
 
 #include "libavutil/common.h"
 #include "libavutil/x86/asm.h"
diff --git a/libavutil/arm/intmath.h b/libavutil/arm/intmath.h
index f19b21e98d..96b4ad4fe3 100644
--- a/libavutil/arm/intmath.h
+++ b/libavutil/arm/intmath.h
@@ -23,7 +23,7 @@
 
 #include <stdint.h>
 
-#include "config.h"
+#include "cbs/config.h"
 #include "libavutil/attributes.h"
 
 #if HAVE_INLINE_ASM
diff --git a/libavutil/intmath.h b/libavutil/intmath.h
index 52e11a8d5f..ae69932edf 100644
--- a/libavutil/intmath.h
+++ b/libavutil/intmath.h
@@ -23,7 +23,7 @@
 
 #include <stdint.h>
 
-#include "config.h"
+#include "cbs/config.h"
 #include "attributes.h"
 
 #if ARCH_ARM
diff --git a/libavutil/riscv/intmath.h b/libavutil/riscv/intmath.h
index a09248f903..a93c646594 100644
--- a/libavutil/riscv/intmath.h
+++ b/libavutil/riscv/intmath.h
@@ -24,7 +24,7 @@
 #include <stdint.h>
 #include <math.h>
 
-#include "config.h"
+#include "cbs/config.h"
 #include "libavutil/attributes.h"
 #include "libavutil/riscv/cpu.h"
 
diff --git a/libavutil/x86/asm.h b/libavutil/x86/asm.h
index 9bff42d628..6977f019bc 100644
--- a/libavutil/x86/asm.h
+++ b/libavutil/x86/asm.h
@@ -22,7 +22,7 @@
 #define AVUTIL_X86_ASM_H
 
 #include <stdint.h>
-#include "config.h"
+#include "cbs/config.h"
 
 typedef struct xmm_reg { uint64_t a, b; } xmm_reg;
 typedef struct ymm_reg { uint64_t a, b, c, d; } ymm_reg;
diff --git a/libavutil/x86/intmath.h b/libavutil/x86/intmath.h
index 4893a1f1b4..ac259e6335 100644
--- a/libavutil/x86/intmath.h
+++ b/libavutil/x86/intmath.h
@@ -30,7 +30,7 @@
 #include <immintrin.h>
 #endif
 #endif
-#include "config.h"
+#include "cbs/config.h"
 
 #if HAVE_FAST_CLZ
 #if (defined(__INTEL_COMPILER) && (__INTEL_COMPILER>=1216)) || defined(_MSC_VER)
